llvm/tools/llvm-mca/Views/InstructionInfoView.cpp

```diff
@@ -250,7 +239,7 @@ void InstructionInfoView::collectData(
       raw_string_ostream TempStream(IIVDEntry.Resources);
       const MCWriteProcResEntry *Index = STI.getWriteProcResBegin(&SCDesc);
       const MCWriteProcResEntry *Last = STI.getWriteProcResEnd(&SCDesc);
-      auto Sep = "";
+      StringRef Sep = "";
```

Comment:
Reading your code again, I think you can use `ListSeparator` here: https://github.com/llvm/llvm-project/blob/617646ab8d148ab51e8b94ccc0a40af2fc6c3dcf/llvm/include/llvm/ADT/StringExtras.h#L527